Dockerfile

```
@@ -0,0 +1,23 @@
+# app/Dockerfile
+
+FROM python:3.7-slim
+
+EXPOSE ${PORT}
+
+WORKDIR /app
+
+RUN apt-get update && apt-get install -y \
+  build-essential \
+  ffmpeg \
+  libsm6 \
+  libxext6 \
+  curl \
+  software-properties-common \
+  git \
+  && rm -rf /var/lib/apt/lists/*
+
+RUN git clone https://github.com/wiktorlazarski/head-segmentation.git .
+
+RUN pip3 install -e .
+
+ENTRYPOINT exec python -m streamlit run apps/web_checking.py --server.port=$PORT --server.address=0.0.0.0
```

Just a question: would that be possible to make this Dockerfile slightly more generic by parametrising Streamlit app that should be started? I'm thinking about structure that looks something like:
```suggestion
ENTRYPOINT exec python -m streamlit run apps/$APP_SCRIPT --server.port=$PORT --server.address=0.0.0.0
```
wdyt?

---

That is a very nice idea! However, currently there is absolutely no need for that since we ONLY want `web_checking.py` to run in the cloud and this change would force us to reconfigure current cloud deployment which I'm against.

---

It would make sense if we'd have another 'deployable' version e.g backend API (we can think about it in the future)